.github/workflows/test_on_deploy.yml

```diff
@@ -0,0 +1,32 @@
+name: Run Integration Tests on Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9, 3.10, 3.11]
```

Comment:
3.10 need to be put in qoutes [example](https://github.com/mindsdb/minds_python_sdk/blob/e222e5df71f703eb27cf966af05a443ecd0de206/.github/workflows/test_on_pr.yml#L14)